spec/minimax_server/server_configuration_spec.clj

```diff
@@ -0,0 +1,18 @@
+(ns minimax-server.server-configuration-spec
+  (:import [scarvill.httpserver.server ServerConfiguration HttpService])
+  (:require [speclj.core :refer :all]
+            [minimax-server.server-configuration :refer :all]))
+
+(describe "server configuration"
+
+  (it "implements ServerConfiguration"
+      (should-be-a ServerConfiguration (minimax-server-config 5000 identity)))
+
+  (it "returns port given at init"
+      (should= 5000 (.getPort (minimax-server-config 5000 identity))))
+
+  (it "returns nil for public directory"
+      (should= nil (.getPublicDirectory (minimax-server-config 5000 identity))))
+
+  (it "returns an HttpService"
+      (should-be-a HttpService (.getService (minimax-server-config 5000 identity)))))
```

Comment:
I would just delete this file. What behavior is desired from the return value being an `HttpService`?